index.js

```diff
@@ -18,6 +18,30 @@ var getAllAccountsData = (db, callback) => {
     })
 }
 
+var getAccountsBelowBal = (db, upperBal, callback) => {
+    var collection = db.collection('accounts')
+    collection.find({balance: {$lt: upperBal}}).toArray((err, docs) => {
+        results = []
+        docs.forEach(doc => {
+            doc.balance = doc.balance.toString()
+            results.push(doc)
+        })
```

Comment:
I'm not sure you need these? My code doesn't have any of these lines, it just goes straight to 'callback(docs)'

Reply:
This was a fix to stop Postman app treating the decimal like it was an object